### ISSUE TITLE
OpenBLAS detection within math ?

### ISSUE THREAD

**Issue by miroi** [coderefinery/autocmake#172]
Hi,

I see gotoblas cmake script among math detection tool; how about the [OpenBLAS](https://github.com/xianyi/OpenBLAS) detection ?

**Comment by bast** [comment]
Would be good to have. Feel free to contribute. But please note that soon the `yaml` branch will merge to master and the `autocmake.cfg` has changed to `autocmake.yml`.

**Comment by miroi** [comment]
Hi Rado,

the problem is that libopenblas packages are conflicting with installed blas/lapack/atlas packages.

what to do ?

One solution would be setting up extra virtual machine in travis that has only libopenblas library installed and there we would test the libopenblas only (with fc, let say).  

Comemnts ?

**Comment by bast** [comment]
Yes I would try to do this with different jobs under Travis. This should be possible. If it is too difficult, then we don't test it within Travis and we let the community test it.

**Comment by miroi** [comment]
Hi,

while working on it, i got ideas: i) own new test for openblas,  test_openblas.py; ii) extended travis-ci.yml for extra linux configuration with libopenblas and co, with running the  test_openblas.py .

Is shoud be possible to have more testing segments in travis-ci, with each different installed packages ?

**Comment by bast** [comment]
I think this should be possible.

**Comment by miroi** [comment]
good, going to try it , with God's help ....

**Comment by miroi** [comment]
I asked on travis support, hopefully they will advice me, how to arrange .travis-ci.yml so that it can handle multiple runs with different installed packages and different test scripts.
